### PR TITLE
Install/Setup content

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -7,3 +7,472 @@ endif::[]
 Install API Community Manager with these instructions.
 
 Installed as a Salesforce package.
+
+API Community Manager 
+Setup Guide
+ (RC Version)
+
+Organization Installation Requirements - prereqs???
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this should go on the pre-reqs section

File: modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Installing API Community Manager 
+Update API Community Manager - move this???
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this should go on a new page `updating.adoc` (I've added the link to the nav doc in the rigth place but haven't created the page)

File: modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Create a Community
+API Community Manager Components
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this is already on a different section of docs - you can take it out here

File: modules/ROOT/pages/installation.adoc [hunk 4]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Publish API Community Manager
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: remove this from here - instead, add a section "Customizing the look&feel and publishing your community" that just points the reader to the `brand-intro.adoc` file (link is in nav but didn't create the file yet).

File: modules/ROOT/pages/installation.adoc [hunk 5]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Configure the Guest User Access
+Configure the Registered Users
+Configure Api Community Manager
+Set permissions for Unauthenticated users
+How to enable external users to register in your Community
+Define moderation rules for forums
+How to setup an Approval Process for new Community Users
+Customize Login and Registration Pages
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this should be in the branding and cusotmization section, so just remove it from here

File: modules/ROOT/pages/installation.adoc [hunk 6]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Troubleshooting
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this should go in a separated page `troubleshooting.adoc` - remove from here

File: modules/ROOT/pages/installation.adoc [hunk 7]
```diff
@@ -7,3 +7,472 @@ endif::[]
...
+Reference: Auth Provider details
```
Comment (jdeoliveira) [inline modules/ROOT/pages/installation.adoc]: this should go on a separated page `reference.adoc` - remove from here

**Review by jdeoliveira — DISMISSED** [review]

File: modules/ROOT/pages/installation.adoc (hunk 7 above)
Comment (paigehargrave) [reply modules/ROOT/pages/installation.adoc]: I've removed it - thank you!

File: modules/ROOT/pages/installation.adoc (hunk 6 above)
Comment (paigehargrave) [reply modules/ROOT/pages/installation.adoc]: I've removed this - thank you!

File: modules/ROOT/pages/installation.adoc (hunk 5 above)
Comment (paigehargrave) [reply modules/ROOT/pages/installation.adoc]: Removed.

File: modules/ROOT/pages/installation.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/installation.adoc]: Removed - thank you!

File: modules/ROOT/pages/installation.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/installation.adoc]: Removed it - thanks!

**Comment by wwarner-salesforce** [comment]
@jdeoliveira , should this install page contain the link `https://login.salesforce.com/packaging/installPackage.apexp?p0=04t4P000002ErbpQAC`, or say "the package installation link provided by your MuleSoft representative"? (I think we do not want the link in the docs, but I want to be sure.)

**Comment by paigehargrave** [comment]
PTAL at the 'Communities' and 'Users' sections - I think the headings should be nested (they appear to be at the same level as the overall section heading), but I'll defer to you. Thanks!

**Comment by wwarner-salesforce** [comment]
'Communities' and 'Users' should be at level 2, under 'Configure API Community Manager', then? In that case the document is 4 levels deep, and lines 49-101 have the first two sections, while lines 103 through 404 are part of the third section. This is a deeper and more uneven division of a file than we usually have, but it is long and complex material, and the content may dictate that organization pattern. As long as you're sure, it's good with me.

**Comment by paigehargrave** [comment]
Added the additional nesting to avoid confusion.

**Comment by paigehargrave** [comment]
@wwarner-salesforce - PTAL - thank you!

**Comment by wwarner-salesforce** [comment]
Looks great! I built locally and am making changes now to fix formatting errors.

**Comment by paigehargrave** [comment]
I think line 272 is a bit redundant - do we need an intro sentence (I think the title is pretty clear, but perhaps our style guidelines dictate this)?

**Comment by wwarner-salesforce** [comment]
I do always try to have some text in each section before the subsections start, but I don't think the style guide dictates it, so I'm ok with removing 272.

**Comment by wwarner-salesforce** [comment]
I just did a local build to check, and the link "Enable Communities Inside Your Organization" causes the new tab to pop open when you click that item in the TOC, so I'm going to modify that slightly.

**Comment by wwarner-salesforce** [comment]
Jesus confirmed today that we should refer customers to their Salesforce rep and not link to the installation package.

I'm removing line 272.

Currently we have "Set Up Your API Community" with no content and two sub-pages: "Installation Pre-requisites" and "Installation". The prerequisites are a tiny amount of info. So "Installation" already has almost all the info on these three pages, and I'm fixing that confusing structure by replacing the three with the one "Installation" page. I'm happy to modify that later if needed.

**Comment by wwarner-salesforce** [comment]
I'm going to go ahead and merge this PR with the manual line numbers on certain section headers, but I look forward to moving to the better `:sectnums:` and `:sectnums!:` system once Fernando approves.

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**